src/Common/Http/Request.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * SocialConnect project
+ * @author: Patsura Dmitry @ovr <talk@dmtry.me>
+ */
+declare(strict_types=1);
+
+namespace SocialConnect\Common\Http;
+
+class Request
```

Comment:
Since you are now using a PSR-7 implementation `guzzlehttp/psr7`, I am curious why having this request class is necessary? All the methods it has are essentially already provided by `ServerRequestInterface`.

`SocialConnect\OAuth1\Request` does have extra methods but it could directly extend the server request class provided by `guzzlehttp/psr7` instead of this one. Then the `SocialConnect\OAuth1\Request` instance could be directly passed to http client.

Reply:
@ADmad Good catch, didn't remove it because I am still thinking
Should I implement own `PSR7` message or continue to use `guzzlehttp/psr7 `

> SocialConnect\OAuth1\Request does have extra methods but it could directly extend the server request class provided by guzzlehttp/psr7 instead of this one. Then the SocialConnect\OAuth1\Request instance could be directly passed to http client.

It's rly cool idea 👍 Will implemented this now

Reply:
Implemented https://github.com/SocialConnect/auth/pull/84/commits/f37e94a14e07505e1fd14e5b6c228631eb5a4eae